js/parallax.js

```diff
@@ -39,7 +39,7 @@
         $this.children("img").one("load", function() {
           updateParallax(true);
         }).each(function() {
-          if(this.complete) $(this).load();
+          if(this.complete) updateParallax(true);
```

Comment:
Sry, I missed the comment. I will test it myself and will give feedback and accordingly a commit.